.github/CODEOWNERS

```diff
@@ -80,4 +80,8 @@
 /styles/wikipedia @GitMuslim
 /styles/wikiwand @Tnixc
 /styles/youtube @isabelroses @uncenter
-/styles/nextjs @Dandraghas
\ No newline at end of file
+/styles/nextjs @Dandraghas
+
+.github/CODEOWNERS @catppuccin/userstyles-staff
```

Comment:
This file is auto-generated but it's good practice to have a code owner against the file itself.